### DISCUSSION TITLE
I have some questions about Uno

### DISCUSSION THREAD

**Discussion by brunofpa** [unoplatform/uno#5835] [Q&A]
Hello, I'm developing a application that needs to be targeted for Web, Android and iOS. I was looking into using React.js for Web and Xamarin Forms for Android and iOS. But I thought that it should be much more easier if I just use Uno to target all those platforms. But I have some questions:

1. What is the performance in Uno WebAssembly compared to React.js?
2. Does Uno have support for animations and transitions?
3. Does Uno have consistent animations between all platforms? Eg. GridView animation when adding items
4. If Uno won't have animations or transitions, am I able to implement in other platforms separately?
5. Are there some web apps in production with Uno?
6. I have tried the XAML Controls Gallery web app in https://xamlcontrolsgallery.platform.uno/ and the Calculator in https://calculator.platform.uno/ and it takes a while to load the website, is this a common behavior in Uno apps?

That's a lot of questions, but I need to be sure that Uno is fast and fluid for my app. Thanks

**Comment by sasakrsmanovic** [comment]
Hi @brunofpa , 

Thanks for writing and for considering Uno. Perhaps the fastest way to answer your question would be via a quick call; there are likely follow-up questions you'd ask. We also have questions about your solution to judge what kind of fit is Uno for you. The short answer is  that we have many customers using Uno in the scenario you are looking into - Web + Mobile., in LOB scenarios. We have covered the best use cases in this blog post - https://platform.uno/blog/top-5-webassembly-use-cases-for-net-developers/.  If you'd like, please reach out to sasha @ platform.uno and I will schedule a call with our tech team. Thanks.
